core/docker/container-test.sh

```diff
@@ -51,7 +56,7 @@ function test_container {
     local CONTAINER_NAME=$1
     local PLATFORM=$2
     echo "🐢 Validating ${CONTAINER_NAME} on platform ${PLATFORM}..."
-    test_javahome ${CONTAINER_NAME} ${PLATFORM}
-    test_trino_starts ${CONTAINER_NAME} ${PLATFORM}
+    test_javahome "${CONTAINER_NAME}" "${PLATFORM}"
+    test_trino_starts "${CONTAINER_NAME}" "${PLATFORM}"
```

Comment:
separate commit

Reply:
Done! Sorry, I keep forgetting mixing formatting changes like this makes reviews harder. I should probably be doing more reviews myself :-)